# translation of de.po to German
# translation of de.po to
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Mustafa Ölcerman <oelcerman@fussballkreis.org>, 2006.
# Mustafa Ölcerman <oelcerman@fussballkreis.org>, 2007.
# Mustafa Ölcerman <mustafa.oelcerman@fk-koeln.de>, 2007.
# Mustafa Ölcerman <m.oelcerman@pardus-linux.de>, 2007, 2008.
# Mustafa Ölcerman <m.oelcerman@pardus-linux.eu>, 2009.
# Klemens Häckel <click3d at linuxmail dot org>, 2009.
msgid ""
msgstr ""
"Project-Id-Version: de\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-12-08 12:05+0200\n"
"PO-Revision-Date: 2009-06-26 21:34-0400\n"
"Last-Translator: Klemens Häckel <click3d at linuxmail dot org>\n"
"Language-Team: German <es@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: KBabel 1.11.4\n"

#: bin/mudur.py:227
msgid "Cannot write mudur.log, read-only file system"
msgstr "Невозможно записать mudur.log, файловой система только для чтения"

#: bin/mudur.py:395
msgid "Safe Mode"
msgstr "Безопасный режим"

#: bin/mudur.py:399
msgid "Cannot find /etc/pardus-release"
msgstr "Не могу найти /etc/pardus-release"

#: bin/mudur.py:492
msgid "/etc/env.d/03locale cannot be updated"
msgstr "/etc/env.d/03locale не может быть обновлен"

#: bin/mudur.py:518
#, python-format
msgid "Could not set unicode mode on tty %d"
msgstr "Не удалось установить режим Юникод на tty %d"

#: bin/mudur.py:576
msgid "Cannot connect to DBus, services won't be started"
msgstr "Не удается подключиться к DBus, сервис не будет запущен"

#: bin/mudur.py:592
#, python-format
msgid ""
"Unable to start network:\n"
"  %s"
msgstr ""
"Не удается запустить сеть::\n"
"  %s"

#: bin/mudur.py:597
msgid "Cannot start system logger"
msgstr "Не удается запустить службу журналирования системы"

#: bin/mudur.py:603
msgid "Starting services"
msgstr "Запуск служб"

#: bin/mudur.py:636
msgid "Stopping services"
msgstr "Остановка служб"

#: bin/mudur.py:664 bin/service.py:161 bin/service.py:270
#, python-format
msgid "Starting %s"
msgstr "Запуск %s"

#: bin/mudur.py:675 bin/service.py:174 bin/service.py:282
#, python-format
msgid "Stopping %s"
msgstr "Остановка %s"

#: bin/mudur.py:692
msgid "Triggering udev events which are failed during a previous run"
msgstr "Запуск Udev событий, которые не удалось запустить во время предыдущего запуска"

#: bin/mudur.py:716
msgid "Can't move persistent udev rules from /dev/.udev"
msgstr "Не могу установить правила udev из /dev/.udev"

#: bin/mudur.py:728
msgid "Starting udev"
msgstr "Запуск udev"

#: bin/mudur.py:742
msgid "Populating /dev"
msgstr "Запись /dev"

#: bin/mudur.py:768
msgid "Removing stale lock file /etc/mtab~"
msgstr "Удаление устаревшего файла блокировки  /etc/mtab~"

#: bin/mudur.py:771
msgid "Failed removing stale lock file /etc/mtab~"
msgstr "Не удалось удалить устаревший файл блокировки /etc/mtab~"

#: bin/mudur.py:783
msgid "/etc/fstab doesn't contain an entry for the root filesystem"
msgstr "/etc/fstab не содержит записи для корневой файловой системы"

#: bin/mudur.py:790
msgid "Remounting root filesystem read-only"
msgstr "Перемонтирование корневая файловая системы только для чтения"

#: bin/mudur.py:796
msgid "Checking root filesystem (full check forced)"
msgstr "Проверка корневой файловой системы (полная проверка принудительно)"

#: bin/mudur.py:802
msgid "Checking root filesystem"
msgstr "Проверка корневой файловой системы"

#: bin/mudur.py:811
msgid "Filesystem repaired, but reboot needed!"
msgstr "Файловая система отремонтирована, но необходима перезагрузка!"

#: bin/mudur.py:815
msgid "Rebooting in 10 seconds..."
msgstr "Перезагрузка через 10 секунд..."

#: bin/mudur.py:817
msgid "Rebooting..."
msgstr "Перезагрузка..."

#: bin/mudur.py:821
msgid "Filesystem could not be repaired"
msgstr "Файловая система не может быть восстановлена"

#: bin/mudur.py:826
msgid "Skipping root filesystem check (fstab's passno == 0)"
msgstr "Пропуск проверки корневой файловой системы (fstab's passno == 0)"

#: bin/mudur.py:834
msgid "Remounting root filesystem read/write"
msgstr "Перемонтирование корневой файловой системы для чтения/записи"

#: bin/mudur.py:838
msgid ""
"Root filesystem could not be mounted read/write\n"
"   You can either login below and manually check your filesytem(s) OR\n"
"   restart your system, press F3 and select 'FS check' from boot menu\n"
msgstr ""
"Корневая файловая система не может быть смонтирована для чтения и записи\n"
"Вы можете войти ниже и вручную проверить файловую систему(ы) или\n"
"перезагрузить систему, нажмите F3 и выберите 'FS check' из меню загрузки\n"

#: bin/mudur.py:850
msgid "Couldn't synchronize /etc/mtab from /proc/mounts"
msgstr "Не удалось произвести синхронизацию /etc/mtab с /proc/mounts"

#: bin/mudur.py:870
msgid "Checking all filesystems"
msgstr "Проверка всех файловых систем"

#: bin/mudur.py:874
msgid "A full fsck has been forced"
msgstr "Полный FSCK был остановлен"

#: bin/mudur.py:892
msgid "Filesystem errors corrected"
msgstr "Ошибки файловой системы исправлены"

#: bin/mudur.py:894
msgid "Fsck could not correct all errors, manual repair needed"
msgstr "Fsck не может исправить все ошибки автоматически, нужно ручное исправление"

#: bin/mudur.py:902
msgid "Mounting local filesystems"
msgstr "Монтаж локальных файловых систем"

#: bin/mudur.py:911
msgid "Mounting remote filesystems"
msgstr "Монтирования удаленных файловых систем"

#: bin/mudur.py:963
#, python-format
msgid "Setting up hostname as '%s'"
msgstr "Настройка хоста '%s'"

#: bin/mudur.py:986
msgid "Setting disk parameters"
msgstr "Настройка параметров диска"

#: bin/mudur.py:1007
msgid "Activating swap space"
msgstr "Активация подкачки"

#: bin/mudur.py:1018
msgid "Deactivating swap space"
msgstr "Отключение подкачки"

#: bin/mudur.py:1028
msgid "Cleaning up /var"
msgstr "Очистка /var"

#: bin/mudur.py:1044
msgid "Cleaning up /tmp"
msgstr "Очистка /tmp"

#: bin/mudur.py:1081
msgid "Setting system clock to hardware clock"
msgstr "Настройка системных часов"

#: bin/mudur.py:1097
msgid "Failed to adjust systematic drift of the hardware clock"
msgstr "Не удалось настроить систематическую отправку из аппаратных часов"

#: bin/mudur.py:1101
msgid "Failed to set system clock to hardware clock"
msgstr "Не удалось установить системные часы"

#: bin/mudur.py:1112
msgid "Syncing system clock to hardware clock"
msgstr "Синхронизация системного времени"

#: bin/mudur.py:1115
msgid "Failed to synchronize clocks"
msgstr "Ошибка синхронизации системного времени"

#: bin/mudur.py:1172
msgid "Unmounting filesystems"
msgstr "Отключение файловых систем"

#: bin/mudur.py:1180
msgid "Remounting remaining filesystems read-only"
msgstr "Перемонтирование остальных файловых систем только для чтения"

#: bin/mudur.py:1195
msgid "An internal error occured. Please report to the bugs.pardus.org.trwith following information:"
msgstr "Произошла внутренняя ошибка. Пожалуйста сообщите по адресу bugs.pardus.org.tr следующую информацию:"

#: bin/mudur.py:1296
msgid "Setting up localhost"
msgstr "Настройка localhost"

#: bin/mudur.py:1309
msgid "Updating environment variables"
msgstr "Обновление переменных среды"

#: bin/mudur.py:1361
msgid "Trying to initiate a warm reboot (skipping BIOS with kexec kernel)"
msgstr "Попытка инициирования теплой перезагрузки (пропуск BIOS с Kexec ядро)"

#: bin/service.py:60
msgid "local"
msgstr "local"

#: bin/service.py:61
msgid "script"
msgstr "скрипт"

#: bin/service.py:62
msgid "server"
msgstr "сервер"

#: bin/service.py:74
msgid "running"
msgstr "запущен"

#: bin/service.py:76
msgid "yes"
msgstr "да"

#: bin/service.py:78
msgid "conditional"
msgstr "conditional"

#: bin/service.py:101
msgid "Status"
msgstr "Статус"

#: bin/service.py:102
msgid "Service"
msgstr "Служба"

#: bin/service.py:103
msgid "Autostart"
msgstr "Автозапуск"

#: bin/service.py:104
msgid "Description"
msgstr "Описание"

#: bin/service.py:147 bin/service.py:157
#, python-format
msgid "Unable to start %s:"
msgstr "Не удается запустить %s:"

#: bin/service.py:170
#, python-format
msgid "Unable to stop %s:"
msgstr "Не удается остановить %s:"

#: bin/service.py:183
#, python-format
msgid "Unable to set %s state:"
msgstr "Невозможно установить состояние %s :"

#: bin/service.py:188
#, python-format
msgid "Service '%s' will be auto started."
msgstr "Служба '%s' будет запущена автоматически."

#: bin/service.py:190
#, python-format
msgid "Service '%s' won't be auto started."
msgstr "Служба '%s'  не будет  запущена автоматически."

#: bin/service.py:192
#, python-format
msgid "Service '%s' will be started if required."
msgstr "Служба '%s'  будет запущена, если потребуется."

#: bin/service.py:201
#, python-format
msgid "Unable to reload %s:"
msgstr "Невозможно перезапустить %s:"

#: bin/service.py:205
#, python-format
msgid "Reloading %s"
msgstr "Перезапуск %s"

#: bin/service.py:259
msgid "You must be root to use that."
msgstr "Вы должны быть суперпользователем, чтобы использовать это."

#: bin/service.py:278
msgid "Unable to start DBus"
msgstr "Не удается запустить DBus"

#: bin/service.py:292
msgid "DBus is not running."
msgstr "DBus не запущен."

#: bin/service.py:294
msgid "DBus is running."
msgstr "DBus запущен."

#: bin/service.py:299
msgid ""
"usage: service [<options>] [<service>] <command>\n"
"where command is:\n"
" list     Display service list\n"
" status   Display service status\n"
" info     Display service status\n"
" on       Auto start the service\n"
" off      Don't auto start the service\n"
" start    Start the service\n"
" stop     Stop the service\n"
" restart  Stop the service, then start again\n"
" reload   Reload the configuration (if service supports this)\n"
"and option is:\n"
" -N, --no-color  Don't use color in output\n"
" -q, --quiet     Don't print replies"
msgstr ""
"Использование: service [<options>] [<service>] <command>\n"
"where command is:\n"
" list Display service list\n"
" status Display service status\n"
" info Display service status\n"
" on Auto start the service\n"
" off Don't auto start the service\n"
" start Start the service\n"
" stop Stop the service\n"
" restart Stop the service, then start again\n"
" reload Reload the configuration (if service supports this)\n"
"and option is:\n"
" -N, --no-color Don't use color in output\n"
" -q, --quiet Don't print replies"

#: bin/service.py:355
#, python-format
msgid "No such service: %s"
msgstr "Нет такой службы: %s"

#~ msgid "Networking backend is set to NetworkManager"
#~ msgstr "NetworkManager выбран как бэкэнд для работы с сетью"

#~ msgid "Bringing up %s"
#~ msgstr "Bringing up %s"

#~ msgid "Unable to bring up %s"
#~ msgstr "Unable to bring up %s"

#~ msgid "No network connection, skipping remote mount."
#~ msgstr "Нет подключения к сети, пропуск удаленного монтирования"

#~ msgid "Mounting remote filesystems (CTRL-C stops trying)"
#~ msgstr "Монтирования удаленных файловых систем (CTRL-C прекращает попытки)"

#~ msgid "Mounting skipped with CTRL-C, remote shares will not be accessible!"
#~ msgstr "Монтирование отменено!"

#~ msgid "Mounting USB filesystem"
#~ msgstr "Монтирование USB файловой системы"

#~ msgid "Starting portmap service for NFS"
#~ msgstr "Запуск Portmap службы для NFS"

#~ msgid "Mounting /dev"
#~ msgstr "/dev wird gemountet"

#~ msgid "Mounting /sys"
#~ msgstr "/sys wird gemountet"

#~ msgid "Mounting /dev/pts"
#~ msgstr "/dev/pts wird gemountet"

#~ msgid "Root filesystem could not be mounted read/write"
#~ msgstr "Root Dateisystem konnte nicht mit Lese-/Schreibzugriff gemountet werden"
